### PR TITLE
Fixes #36 - Tagging and branching strategy

### LINKED ISSUE #36: Tagging and branching strategy
We should define tagging and branching strategy 

**Here is initial proposal for tagging:**
When:
 - Tag new major version when new WF major is released and covers new MP related features

How:
 - Use https://maven.apache.org/maven-release/maven-release-plugin/

Naming:
 - Same as WildFly version (what if there is WF release without MP related feature ?)
 - Start with 1.0.0.Final (we can have CR1/CR2)
 - Start with 1.0.0

**Here is initial proposal for branching:**
 - No branches unless really needed, tags should be enough
 - If branch is needed its name will be $MAJOR.$MINOR, e.g. 1.0 for 1.0.z releases

### PR DESCRIPTION
**Pull request by istraka**
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- N/A Link to the passing job is provided: no change in code
- N/A Code is self-descriptive and/or documented: no change in such matter
- N/A Description of the tests scenarios is included (see #46): no new tests are introduced

### CODE REVIEW

**Comment by mnovak1** [comment]
Thanks, merging.

File: pom.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (rsvoboda) [inline pom.xml]: this was unnecessary change, next time @mnovak1 please ask for revert of such things.

Other pom.xml files have 2 line definition for better readability
@istraka / @mnovak1 please prepare pr to revert this line change
